SpatialGDK/Source/SpatialGDK/Public/Utils/SpatialStatics.h

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "Kismet/BlueprintFunctionLibrary.h"
+#include "SpatialStatics.generated.h"
+
+UCLASS()
+class SPATIALGDK_API USpatialStatics : public UBlueprintFunctionLibrary
+{
+    GENERATED_BODY()
+
+private:
+	
+	static bool IsSpatialOffloadingEnabled();
+
+	static class UActorGroupManager* GetActorGroupManager(const UObject* WorldContext);
+
+public:
+
+	/**
+	 * Returns true if SpatialOS Networking is enabled.
+	 */
+	UFUNCTION(BlueprintPure, Category = "SpatialOS")
+	static bool IsSpatialNetworkingEnabled();
+
+	/**
+     * Returns true if the current Worker Type owns the Actor Group this Actor belongs to.
```

Comment:
Whitespace seems off, make sure you have tabs rather than spaces here for the whitespaces

Reply:
Actually may just be indentation being wrong

Reply:
Done